public/scripts/editor/js/bramble-editor.js

```diff
@@ -38,12 +40,19 @@ define(function(require) {
       });
 
       // Start loading the project files
-      Project.load(makeDetails, host, authenticated, function(err, fileToOpen) {
+      Project.load(project, host, function(err, fileToOpen) {
         if(err) {
           console.error("[Thimble Error]", err);
           return;
         }
 
+        // Initialize the project name UI
+        title = project.title || Project.getAnonymousTitle();
+        projectNameComponent = new ProjectRenameUtility(host, authenticated, csrfToken, title);
```

Comment:
ProjectRenameUtility should just use the `Project` module directly, and we should keep the getter/setter for `title` there.